tools/ClassGen/InstrGen.cpp

```diff
@@ -138,6 +140,7 @@ int main(int argc, char **argv) {
       .addMember(MemberType::VectorUnsigned, "Kernels")
       .addMember(MemberType::VectorUnsigned, "Strides")
       .addMember(MemberType::VectorUnsigned, "Pads")
+      .addMember(MemberType::Unsigned, "Layout")
```

Comment:
why not enum as well?

Reply:
Enums are not supported for Instructions (enum is basically a typedef of Unsigned anyway in the Nodegen so no real difference).